tests/playwright/deployment-tests/health.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test } from '@playwright/test'
+import { HealthPage } from '../page-object-model/health-page'
+
+test.describe('Health status endpoint', () => {
+  let healthPage: HealthPage
+  test.beforeEach(({ page }) => {
+    healthPage = new HealthPage(page)
+  })
+
+  test('User can navigate to the health endpoint', async () => {
+    await healthPage.goTo()
+    await healthPage.expect.toHaveStateMatching('Healthy')
```

Comment:
We could improve this by moving the `'Healthy'` into the page object model. Our assertion would then be something like `await healthPage.expect.toBeHealthy()` which would then make it easier later down the line if we change what "healthy" looks like